### ISSUE TITLE
Graph matching/similarity feature request

### ISSUE THREAD

**Issue by Sherafgan** [neo4j/neo4j#8624]
Hi there!
Just wondering, are you, guys, considering adding graph matching/similarity feature?

Ex: 
Giving G1 to database, db returns a list of similar graphs comparing values/properties/etc.:
1. G2 (sim: 0.89%)
2. G3 (sim: 0.78%)
3. G4 (sim: 0.71%)
4. G5 (sim: 0.65%)

**Comment by spacecowboy** [comment]
No such plans exist. I guess what you really mean is similarity between results of two queries, where similarity is number of matching nodes and relationships?

**Comment by Sherafgan** [comment]
Yes, kind of. Thank you for response!

**Comment by jexp** [comment]
@Sherafgan do you have any source / implementation link for such an algorithm?

**Comment by Sherafgan** [comment]
Yeah! Here is the link for research paper on this problem [_Graph similarity scoring and matching_](http://www.sciencedirect.com/science/article/pii/S0893965907001012).

I myself wanted to use such a feature for research purposes.

**Comment by hvub** [comment]
Still, no priority has been given to such graph similarity feature(s). Feel free to re-request the feature.
